lib/apiservers/portlayer/swagger.json

```diff
@@ -2408,6 +2686,138 @@
 					}
 				}
 			}
+		},
+		"LoggingJoinConfig": {
+			"type": "object",
+			"required": [
+				"handle"
+			],
+			"properties": {
+				"handle": {
+					"type": "object"
```

Comment:
I think the other  endpoints also need to change to object type so that we end up with interface{} instead of string. Assuming the string ID won't have any meaning when we split the services from each other and start moving full spec around

cc @hickeng